flake.nix

```
@@ -51,13 +51,17 @@
         default =
           let
             toolchain = pkgs.rust-bin.fromRustupToolchainFile ./rust-toolchain.toml;
+            pango = pkgs.pango;
           in
           pkgs.mkShell.override
             {
               stdenv = pkgs.stdenvAdapters.useMoldLinker pkgs.clangStdenv;
             }
             {
-              packages = [ toolchain ];
+              packages = [
+                toolchain
+                pango
+              ];
```

```suggestion
              packages = with pkgs; [
                pango
              ] ++ [ toolchain ];
```
this looks cleaner in my opinion and you don't need this `pango = pkgs.pango` line afterwards in the let-in block.